src/main/osd/osd.h

```diff
@@ -299,8 +299,10 @@ typedef struct osdConfig_s {
     uint8_t logo_on_arming_duration;          // display duration in 0.1s units
     uint8_t camera_frame_width;               // The width of the box for the camera frame element
     uint8_t camera_frame_height;              // The height of the box for the camera frame element
+    bool stat_show_cell_value;
     uint16_t task_frequency;
     uint8_t cms_background_type;              // For supporting devices, determines whether the CMS background is transparent or opaque
+    bool stat_show_cell_value;
```

Comment:
Same as line 302

Reply:
Almost there :smirk:

Reply:
Thanks. After merging, a lot of things are screwed up...

Reply:
Feel the struggle. Having many fights with git - eventually made a [wiki entry](https://github.com/betaflight/betaflight/wiki/Betaflight-Contribution)

Reply:
Yes. Sometimes git is a nice tool, but sometimes it's a horror movie...

Reply:
This should be an `uint8_t` to make sure it has a constant size.

Reply:
Makes sense. I'll change this.